docs/catalogue/cat_cohort-data-manager.md

```diff
@@ -61,18 +68,32 @@ template*](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/D
 sheets. Each sheet corresponds to a table in the Data Catalogue (Figure 1). The columns in the sheet correspond to
 columns in the table concerned. This document describes how to fill out each of the sheets and their columns. A column
 with an asterisk (\*) after its name is mandatory, i.e., it should contain values for the system to accept a data
-upload.
+upload. You can download this 
+[*filled out example*](https://github.com/molgenis/molgenis-emx2/raw/master/docs/resources/Dictionary_testCohort.xlsx) 
+as a reference for filling out the template.
 
 <sup>Note that there is no sheet for *AllSourceVariables*. This table is a generic listing of all variables entered for
-the cohort; it shows *SourceVariables* and *RepeatedVariables* in one table.</sup>
+the cohort; it shows *SourceVariables* and *RepeatedSourceVariables* in one table. </sup>
 
 It is good practice trying out adding a few variables to the template first and see whether your upload succeeds. To
-upload the metadata to the Data Catalogue see the section [Upload metadata](cat_cohort-data-manager.md#upload-metadata)
-to the Data Catalogue.
+upload the metadata to the Data Catalogue see the section [Upload metadata](cat_cohort-data-manager.md#upload-metadata).
 
 ![MOLGENIS tables in cohort catalogue](../img/cat_tables-in-cohort-catalogue.png)
 
-<sup>*Figure 2. Tables in a cohort's or DAP's database in the Data Catalogue.*</sup>
+<sup>*Figure 2. Tables in a cohort's or DAP's database in the Data Catalogue. Note that not all tables are filled out 
+via the templates, some are filled via an online form, see section 
+[Fill out cohort rich metadata](cat_cohort-data-manager.md#fill-out-cohort-rich-metadata).*</sup>
+
+#### *SourceDataDictionary* sheet
+
+This sheet is used to fill out versions of your codebook.
+
+| *Column name* | *Description* |
+| --- | --- |
+| resource \* | Fill out your cohort pid. |
+| version \* | Version of the data dictionary. |
+
+<sup>*Table 1. Description of the columns that can be filled out for SourceDataDictionaries. \* = mandatory*</sup>
 
 #### *SourceTables* sheet
 
```

Comment:
The mandatory columns dataDictionary.resource and dataDictionary.version are not in the sheet SourceTables, for the reader it might be confusion that these are actually residing in SourceDataDictionary